src/lib/tachyon-utils.js

```diff
@@ -389,7 +389,7 @@ async function getEDLModeDevices(ui, showSetupMessage) {
 				break;
 			}
 			if (!messageShown) {
-				const defaultMessage = `Waiting for device to enter system update mode...${os.EOL}`;
+				const defaultMessage = `Reconnect the device and enter system update mode...${os.EOL}`;
```

Comment:
Reconnect indicates to me that you have to unplug an already plugged device. It's not quite right.
```suggestion
				const defaultMessage = `Ensure your device is connected, then put it in system update mode...${os.EOL}`;
```